lib/shoryuken/environment_loader.rb

```diff
@@ -15,12 +15,12 @@ def initialize(options)
     end
 
     def load
+      Shoryuken.options.merge!(config_file_options)
+      Shoryuken.options.merge!(options)
```

Comment:
@h3poteto in continuation of https://github.com/phstc/shoryuken/pull/181#issuecomment-208650924. [This](https://github.com/phstc/shoryuken/blob/crowdworks-fix-initialize-logger/lib/shoryuken/environment_loader.rb#L37) will logger out in the `stdout`, as the log wasn't initialized yet. But thinking more on it, I believe it should show an error and exit if the config file supplied is invalid. WDYT?